### PR TITLE
Change default jvm.config to be 16G rather than 2G

### PR DESCRIPTION
**Pull request by cawallin**
Our recommendations for Presto memory are to use 16GB, so we need to
change presto-admin to do that.

### CODE REVIEW

**Comment by cawallin** [comment]
FYI @KBP-TDC

**Comment by ebd2** [comment]
LGTM.

File: docs/presto-admin-commands.rst [hunk 1]
```diff
@@ -113,7 +113,7 @@ configurations:
 *jvm.config* ::
 
     -server
-    -Xmx2G
+    -Xmx16G
```
Comment (akshatnair) [inline docs/presto-admin-commands.rst]: Should we correspondingly also increase the values for query.max_memory_per_node?

Comment (cawallin) [inline docs/presto-admin-commands.rst]: ah yes. probably.

**Pull request merged**